src/funcs/getColumnSettings.js

```diff
@@ -0,0 +1,29 @@
+const getColumnDefault = require('./getColumnDefault.js');
+
+// NOTE: see https://www.dbml.org/docs/#project-definition
+module.exports = function getColumnSettings(col) {
+  const {
+    column_comment: columnComment,
+    column_default: columnDefault,
+    is_nullable: isNullable,
+    dataType,
+    isPrimary
+  } = col;
+
+  const columnSettings = [];
+
+  const primaryKeySetting = isPrimary && `primary key`;
+  if (primaryKeySetting) columnSettings.unshift(primaryKeySetting);
+
+  const defaultSetting = getColumnDefault(columnDefault, dataType);
+  if (defaultSetting) columnSettings.unshift(defaultSetting);
+
+  const notNullSetting = !isNullable && 'not null';
+  if (notNullSetting) columnSettings.unshift(notNullSetting);
+
+  const note = columnComment && `note: '${columnComment}'`;
+  if (note) columnSettings.unshift(note);
```

Comment:
Is the unshift an alternative to using the spread operator?

Reply:
unshift moves it to the from of an array. push puts it at the end.